feat: Automatically generate a Cart ID if non provided (#29)
notrab/react-use-cart: README.md

#### Props

> - `id`: Optional `id` for your cart to help you identify it acrcoss local storage

Reviewer: ```suggestion
- `id`: (_optional_) `id` for your cart to enable automatic cart retrieval via `window.localStorage`
```
